README.md

```diff
@@ -0,0 +1,62 @@
+# package-in-use-detector
+
+A tool for detecting which files are being accessed by running processes in a Linux system using eBPF technology.
+
+## Quick Start
```

Comment:
I would recommend using `nix`. So the command to get dropped into a nix shell with all the dependencies would be 

```sh
nix develop
```